ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -69,6 +74,8 @@
 class ReplicaThread implements Runnable {
 
   private final Map<DataNodeId, List<RemoteReplicaInfo>> replicasToReplicateGroupedByNode;
+  private final Set<PartitionId> replicatedPartitions = new HashSet<>();
```

Comment:
Sorry my intention was "partitionsToReplicate". Anyways, reason why I don't prefer "replicatedPartitions" is that, some of the partitions in the list could have replication disabled. So, lets go with "allPartitions" as we discussed